tests/bplist_regen.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-# Copyright 2018 Arm Limited.
+# Copyright 2018,2020 Arm Limited.
```

Comment:
Being really nitpicky... Any chance you could put a space before `2020` please? :) thanks!

Reply:
Done.